website/docs/r/web_pubsub_hub.html.markdown

```diff
@@ -87,6 +91,18 @@ An `event_handler` block supports the following:
 
 ---
 
+An `event_listener` block supports the following:
+
+* `system_event_name_filter` - (Optional) Specify the list of system events. Supported values are `connect`, `connected` and `disconnected`.
+
+* `user_event_name_filter` - (Optional) Specify the matching event names. There are 3 kind of patterns supported: * `*` matches any event name * `,` Combine multiple events with `,` for example `event1,event2`, it matches event `event1` and `event2` * The single event name, for example `event1`, it matches `event1`. Defaults to `"*"`
```

Comment:
shouldn't this property be a lint then instead of CSV?

Reply:
Thanks kt for the review, sorry that I can't quite follow your question, can you be more specific about it? Thanks

Reply:
sorry list. ie as it is `*` or `event1,event2` it could be user_event_name_filter = ["*"] and user_event_name_filter = ['event1","event2"]